website/docs/guides/service_principal_client_certificate.html.markdown

```diff
@@ -25,7 +25,7 @@ We recommend using either a Service Principal or Managed Service Identity when r
 
 A Service Principal is a security principal within Azure Active Directory which can be granted access to resources within Azure Subscriptions. To authenticate with a Service Principal, you will need to create an Application object within Azure Active Directory, which you will use as a means of authentication, either [using a Client Secret](service_principal_client_secret.html) or a Client Certificate (which is documented in this guide). This can be done using the Azure Portal.
 
-This guide will cover how to generate a client certificate, how to create an Application and linked Service Principal, and then how to assign the Client Certificate to the Application so that it can be used for authentication. Once that's done finally we're going to grant the Service Principal permission to manage resources in the Subscription - to do this we're going to assign `Contributor` rights to the Subscription - however, [it's possible to assign other permissions](https://azure.microsoft.com/en-gb/documentation/articles/role-based-access-built-in-roles/) depending on your configuration.
+This guide will cover how to generate a client certificate, how to create an Application and linked Service Principal, and then how to assign the Client Certificate to the Application so that it can be used for authentication. Once that's done finally we're going to grant the Service Principal permission to manage resources in the Subscription - to do this we're going to assign `Contributor` rights to the Subscription - however, [it's possible to assign other permissions](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles) depending on your configuration.
```

Comment:
I recently learnt that we can safely omit the `en-us` part of the URL, and the link will open with the region/language the user is based in. Just something to consider if you're planning on another round of streamlining the docs.

Reply:
I wasn't, but if you look at my PR history, I am pedantic enough that I will - and it is legitly an improvement.
I'll add it to my queue and create another PR with it down the road.